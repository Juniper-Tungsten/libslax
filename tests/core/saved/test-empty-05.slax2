/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

preserve-space foo goo moo;
strip-space boo coo;

match / {
    <foo> {
        <goo> {
            <moo> "test";
        }
        <boo> {
            <coo> "test";
        }
    }
}
