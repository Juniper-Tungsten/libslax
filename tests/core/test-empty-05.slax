version 1.0;

preserve-space foo goo moo;
strip-space boo coo;

match / {
    <foo> {
	<goo> {
	    <moo> "test";
	}
	<boo> {
	    <coo> "test";
	}
    }
}
