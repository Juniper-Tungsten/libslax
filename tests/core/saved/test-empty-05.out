<?xml version="1.0"?>
<foo>
  <goo>
    <moo>test</moo>
  </goo>
  <boo>
    <coo>test</coo>
  </boo>
</foo>
